theme/sass/components/_buttons.scss

```diff
@@ -0,0 +1,70 @@
+.btn {
+  padding: 1rem 0.8rem;
+  border-radius: 10px;
+
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.btn-small {
+  font-size: $font-xsmall;
+
+  @include breakpoint($md) {
+    font-size: $font-medium;
+  }
+}
+
+.btn-medium {
+  font-size: $font-medium;
+
+  @include breakpoint($md) {
+    font-size: $font-large;
+  }
+}
+
+.btn-large {
+  font-size: $font-large;
+}
+
+.btn-outline {
+  border: 1px solid $primary-color;
+  background-color: white;
+
+  &:hover,
+  &.active {
```

Comment:
Active doesn't belong here, it's not a trait of the button it's more of a donate form trait